docs/integrations/sources/hubspot.md

```diff
@@ -84,12 +96,12 @@ Example of the output message when trying to read `workflows` stream with missin
     "type": "LOG",
     "log": {
         "level": "WARN",
-        "message": 'Stream `workflows` cannot be procced. This hapikey (EXAMPLE_API_KEY) does not have proper permissions! (requires any of [automation-access])'
+        "message": 'Stream `workflows` cannot be proceed. This API key (EXAMPLE_API_KEY) does not have proper permissions! (requires any of [automation-access])'
     }
 }
 ```
 
-### Required scopes
+## Required scopes
 
 If you are using Oauth, most of the streams require the appropriate [scopes](https://legacydocs.hubspot.com/docs/methods/oauth2/initiate-oauth-integration#scopes) enabled for the API account.
```

Comment:
Can we make sure that all instances of `OAuth` are correctly capitalized for both 'O' and 'A'?